portfolio/src/main/webapp/script.js

```diff
@@ -316,3 +412,75 @@ async function deleteAndFetchEmpty() {
     const value = await grabResponse.text();
     document.getElementById("form-container").innerText = value;
 }
+
+//*********************** JSON CONVERSION ***********************
+/**
+ * Creates comment elements from the JSON array
+ * @param {object} jsonArray - JSON array passed in to get information.
+ * @param {number} num - number of comment elements to be created
+ */
+function populateComments(jsonArray, num) {
+    for (var i = 0; i < num; i++) {
+        let value = jsonArray.shift();
+        createCommentElement(value);
+    }
+    numEls = jsonArray.length;
```

Comment:
You should load `numEls` each time until the end. So numEls should probably be:
```
numEls = numEls > jsonArray.length : jsonArray.length : numEls;
```

Reply:
Thank you for this.